cloud/shared/bin/lib/config_loader_test.py

```diff
@@ -2,10 +2,10 @@
 
 from cloud.shared.bin.lib.config_loader import ConfigLoader
 """
- Tests for the ConfigLoader, calls the I/O methods to match the actual
- experience of running the class.
+Tests for the ConfigLoader, calls the I/O methods to match the actual
+experience of running the class.
 
- To run the tests: PYTHONPATH="${PYTHONPATH}:${pwd}" python3 cloud/shared/bin/lib/config_loader_test.py
+To run the tests: PYTHONPATH="${PYTHONPATH}:${pwd}" python3 cloud/shared/bin/lib/config_loader_test.py
```

Comment:
Please add some tests for the code new paths that rely on the server variable definitions, both for the code paths where there are two paths now (e.g. validation) and for new methods (e.g. _load_civiform_server_env_vars()). There may be other, I only picked two examples.